File: app/src/main/java/org/ole/planet/myplanet/utilities/DimenUtils.java
package org.ole.planet.myplanet.utilities;

import android.util.DisplayMetrics;

import org.ole.planet.myplanet.MainApplication;

public class DimenUtils {
    public static int dpToPx(int dp) {
        DisplayMetrics displayMetrics = MainApplication.context.getResources().getDisplayMetrics();
        return Math.round(dp * (displayMetrics.xdpi / DisplayMetrics.DENSITY_DEFAULT));
    }
}

File: app/src/main/java/org/ole/planet/myplanet/utilities/DimenUtils.kt
package org.ole.planet.myplanet.utilities

import android.util.DisplayMetrics
import kotlin.math.roundToInt
import org.ole.planet.myplanet.MainApplication

object DimenUtils {
    @JvmStatic
    fun dpToPx(context: Context, dp: Int): Int {
        val displayMetrics = context.resources.displayMetrics
        return (dp * (displayMetrics.xdpi / DisplayMetrics.DENSITY_DEFAULT)).roundToInt()
    }
}
